sql

```diff
@@ -0,0 +1,46 @@
+create table restoraunt_info(id serial,
+	                         r_name varchar(200) not null,
+							 delivery char(1) not null,
+							 link_to_site varchar(200),
+							 count_of_employers int,
+							 alco char(1),
+							 summer_platform char(1),
```

Comment:
change to "has_summer_platform" and change type to boolean

Reply:
Fixed